apps/api/package.json

```diff
@@ -41,5 +41,10 @@
     "hono": "^4.2.7",
     "superjson": "^2.2.1",
     "zod": "^3.23.5"
+  },
+  "overrides": {
```

Comment:
what does this do?

Reply:
It was intentional - I was having trouble building because I think `rbac` is private. I was attempting to tell `pnpm` I don't have this dependency. This allowed me to `install` cleanly, but not `build` cleanly.

Reply:
Oh, but I can remove this if you like.